framework-plugins/lisk-framework-dashboard-plugin/src/ui/components/Widget/Widget.module.scss

```diff
@@ -0,0 +1,56 @@
+@import '../../styles/constants';
+
+$widget-base-size: 200px;
+
+@mixin widget-size($size) {
+	width: 100%;
+	height: 100px * $size;
+	min-height: 100px * $size;
+}
+
+.root {
+	display: block;
+	margin: 10px 0;
+}
+
+.widget-s {
+	.body {
+		@include widget-size(1);
+	}
+}
+.widget-m {
+	.body {
+		@include widget-size(2);
+	}
+}
+.widget-l {
+	.body {
+		@include widget-size(3);
+	}
+}
+.widget-xl {
+	.body {
+		@include widget-size(4);
+	}
+}
+
+.header {
```

Comment:
thhis is not needed. `Text` component defines the text

Reply:
`.header` defines the wrapper. It can contains `Text` or any other component.

Reply:
im not sure why I said that here 🤔 LGTM